build.gradle

```diff
@@ -26,7 +26,7 @@ ext {
   sizeofVersion = '0.3.0'
 
   // Clustered
-  terracottaPlatformVersion = '5.0.6.beta6'
+  terracottaPlatformVersion = '5.0.6.beta7'
```

Comment:
well, actually I was not talking about just the build.gradle but all changes you made for the compatibility as well
it just feels weird to see 2 upgrades for the same lib in the same PR...